src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -421,6 +421,10 @@ export default {
 		},
 		hasOnePageOfLoans() {
 			return this.totalCount <= this.limit;
+		},
+		showHelpMeChooseFeat() {
+			if (this.enableHelpmeChoose && this.allLoans.length > 8) return true;
+			return false;
```

Comment:
Couldn't this just be `return this.enableHelpmeChoose && this.allLoans.length > 8;` without the if block and second return?

Reply:
oooh yes 🤯
fixed